### ISSUE TITLE
Unable to persist temporary documents on Linux

### ISSUE THREAD

**Issue by nc7s** [VSCodium/vscodium#1937]
**Describe the bug**
When exiting, temporary documents (Untitled-x) trigger a prompt: "Do you want to save the changes you made to (...)? Your changes will be lost if you don't save them." It does when "Don't Save" is chosen.

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/docs/index.md) and [the Troubleshooting page](https://github.com/VSCodium/vscodium/blob/master/docs/troubleshooting.md) and my issue is not mentioned there.

**To Reproduce**
1. Ctrl-N to open a new temporary document
2. Write something
3. Close the application to see the prompt
4. Click "Don't save" and reopen to see it's indeed lost

**Expected behavior**
The temporary document should have persisted, as on Windows and in original VS Code.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Fedora 40
 - Architecture: x86_64 (amd64)
 - Version: 1.90.0, release 24158, commit cc102f3a
 - App Manager: dnf (paulcarroty's repo)
 - Sandboxed: toolbox (podman container)

**Additional context**
The program is run with the following command line:

```
toolbox run --container toolbox-dev codium --enable-features=UseOzonePlatform --ozone-platform=wayland --enable-wayland-ime --new-window
```

**Comment by daiyam** [comment]
Have you tested Visual Studio Code on linux in a podman container?

**Comment by nc7s** [comment]
Yes, just created a toolbox container and installed VS Code official rpm in it, ran the same command. It persisted the document.
